src/bidiMapper/utils/cdpClient.ts

```diff
@@ -17,7 +17,11 @@ import { ServerBinding, AbstractServer } from './iServer';
 import { log } from './log';
 const logCdp = log('cdp');
 
-export class CdpServer extends AbstractServer {
+// TODO: Remove. This is a quick test to verify we can import a node module.
+import { version } from 'devtools-protocol/json/browser_protocol.json';
+logCdp(`Using CDP version: ${version}`);
```

Comment:
When should this be removed?

Reply:
I'll remove on the next PR. I just wanted something in here to use the module to verify it is included in the build output.